core/templates/dev/head/pages/exploration_editor/EditorServices.js

```diff
@@ -629,7 +629,7 @@ oppia.factory('explorationPropertyService', [
       // Validates the given value and returns a boolean stating whether it
       // is valid or not. THIS CAN BE OVERRIDDEN BY SUBCLASSES. The default
       // behavior is to always return true.
-      _isValid: function(value) {
+      _isValid: function() {
```

Comment:
Actually, I agree with @ashish3805. Let's tackle all of this in a separate pass; it's a largish change.